sample.env

```diff
@@ -103,4 +103,8 @@ AEM_LOG_TO_FILE=false
 AEM_LOG_LEVEL=INFO
 ADM_LOG_TO_FILE=false
 ADM_LOG_TO_CONSOLE=true
-ADM_LOG_LEVEL=INFO
\ No newline at end of file
+ADM_LOG_LEVEL=INFO
+
+# ODE Monitoring
+ODE_TIM_INGEST_MONITORING_ENABLED=true
+TIM_INGEST_MONITORING_INTERVAL=60
```

Comment:
Should the interval here be longer if we're setting the default value to one hour in the TimDepositController?

Reply:
I've updated the sample.env file to provide more clarity around what a sane default is in either the local or production scenarios [here](https://github.com/CDOT-CV/jpo-ode/pull/106/commits/516cf8c84cf0331baff4fb6cca36937b066234cc)